src/platform_impl/linux/x11/window.rs

```diff
@@ -242,7 +242,7 @@ impl UnownedWindow {
                 // Find a suitable visual, true color with 32 bits of depth.
                 all_visuals
                     .find_map(|(visual, depth)| {
-                        (visual.class == xproto::VisualClass::TRUE_COLOR)
+                        (depth == 32 && visual.class == xproto::VisualClass::TRUE_COLOR)
```

Comment:
@notgull why do we even search it like that and not by checking the alphamask?

Reply:
"Try to find a 32 bit True Color visual" was [how it was done before](https://github.com/rust-windowing/winit/blob/ff0ce9d065431c93e1d7e5d88bb8cfb15f7f2f60/src/platform_impl/linux/x11/window.rs#L203-L230), and I just ported the logic to `x11rb`. So the real question is "why *not* check the alphamask", which we probably should be doing.